### DISCUSSION TITLE
Filecoin Community Roadmap - 2022Q1 Release

### DISCUSSION THREAD

**Discussion by kaitlin-beegle** [filecoin-project/community#456] [Network + Implementations Announcements]
# Filecoin Community Roadmap 2022Q1
Last month, the Filecoin Community Roadmap project was kicked-off with [an open call for project contributions](https://github.com/filecoin-project/community/discussions/399). 

The purpose was to crowdsource projects that have the potential to significantly impact the Filecoin ecosystem, opening up the network and community to new users and use-cases.  

This information was then used to construct the first iteration of the Community Roadmap: 

![](https://i.imgur.com/AGiGikK.jpg)

## Using the Roadmap

The Roadmap is designed as a tool for community wayfinding; a reference document for everyone.

Its purpose is to provide insight into the strategic development of the network in the coming year and to create a pathway for community members to learn more about ongoing work and connect directly with project teams. 

All of this information can be accessed in the dynamic Roadmap. There, each project block expands to show project details, key project points-of-contact, and links to project websites and developer repos. 

**The dynamic roadmap can be accessed [HERE](https://miro.com/app/board/uXjVOR1oLO8=/?invite_link_id=215641697288).**

For instructions on how to use the dynamic roadmap, check out [THIS VIDEO.](https://www.loom.com/share/403b430965e341f68298839f0c6df8df)

## Next Steps

This is the first iteration of the Filecoin Community Roadmap. 

Each quarter, the Roadmap will be updated.  Existing projects will be audited, new projects will be added, and the structure and design of the Roadmap product will likewise be improved. 

**If you are working on or aware of a project that is not currently featured, you may submit it during the next quarterly update.** The Community Contribution period for 2022Q2 updates will open on March 2, and remain open for one month. 

Again, please note that this is a community project, and is not designed to be inclusive of every project and initative within the Filecoin ecosystem.

**Comment by 876965** [comment]
Is there a roadmap for 2023? or does this encompass 2022-24 and beyond?

**Reply by rjan90** [reply]
Hey @876965! You can check out this Filecoin 2023 Roadmap Presentation here: https://youtu.be/LI3K-BkGitg

**Reply by kaitlin-beegle** [reply]
Hey @876965! We stopped putting maintainer effort into the Community Roadmap because of low community engagement. 

What @rjan90 linked is perfect- a deep dive into Filecoin protocol and product development over the coming years. For a higher level overview, I'd also recommend you check out @jbenet's [Filecoin Masterplan](https://filecoin.io/blog/posts/the-filecoin-masterplan/), which was presented at a recent FIL event. 

Feel free to reach out if you have any further questions! Cheers!
